File: app/src/main/java/com/scatl/uestcbbs/module/message/view/AtMeMsgView.java
package com.scatl.uestcbbs.module.message.view;

import com.scatl.uestcbbs.entity.AtMsgBean;

/**
 * author: sca_tl
 * description:
 * date: 2020/1/29 13:05
 */
public interface AtMeMsgView {
    void onGetAtMeMsgSuccess(AtMsgBean atMsgBean);
    void onGetAtMeMsgError(String msg);
}

File: app/src/main/java/com/scatl/uestcbbs/module/message/view/AtMeMsgView.kt
package com.scatl.uestcbbs.module.message.view

import com.scatl.uestcbbs.base.BaseView
import com.scatl.uestcbbs.entity.AtMsgBean

/**
 * Created by sca_tl at 2023/2/17 14:06
 */
interface AtMeMsgView: BaseView {
    fun onGetAtMeMsgSuccess(atMsgBean: AtMsgBean)
    fun onGetAtMeMsgError(msg: String?)
}